octodns/record/__init__.py

```diff
@@ -219,6 +220,18 @@ def changes(self, other, target):
         if self.ttl != other.ttl:
             return Update(self, other)
 
+    def copy(self, zone=None):
+        _type = getattr(self, '_type')
+        if not self._raw_data.get('type'):
+            self._raw_data['type'] = _type
+
+        return Record.new(
+            zone if zone else self.zone,
+            self.name,
+            self._raw_data,
+            self.source
```

Comment:
I believe we'll want to pass `lenient=True` here so that we copy records that aren't 100% compliant. We don't know how this record was created, it could have had `lenient=True`, so it may not pass validation. If we try to copy that it'd fail. It should be fine to always copy with lenient as whatever is in `self` should be duplicated as-is wether it complies or not.

```suggestion
            self.source,
            lenient=True
```

Reply:
Done.